openmmml/models/utils.py

```diff
@@ -0,0 +1,88 @@
+import torch
+from typing import Tuple
+
+@torch.jit.script
+def simple_nl(positions: torch.Tensor, cell: torch.Tensor, pbc: bool, cutoff: float, sorti: bool=False) -> Tuple[torch.Tensor, torch.Tensor]:
```

Comment:
Do we really need to invent a new neighbor list rather than use the one in NNPOps?